src/v2/perf-utils.js

```diff
@@ -172,17 +172,19 @@ class PUClass {
     const query = { bool: { must: [], must_not: [] }};
     const aggs = { baseline: { percentiles: { field: baselineAgg, percents: [baselineParams.perc] }}};
 
-    // Check the URL
-    // First, remove the trailing '*' -> the Lucene parszer will automatically add it
-    let tmpSearchUrl = baselineParams.searchUrl;
-    if (tmpSearchUrl && tmpSearchUrl.trim().substr(tmpSearchUrl.trim().length - 1) === '*') {
-      tmpSearchUrl = tmpSearchUrl.substring(0, tmpSearchUrl.trim().length - 1);
+    // Decide what URL to filter on
+    this.queryUrl = (baselineParams.searchUrl && baselineParams.searchUrl.indexOf('*') >= 0) ? baselineParams.searchUrl : this.dl;
+    this.queryUrl = this.queryUrl.replace('https://', '').replace('http://', '');
+
+    // Now, remove the trailing '*' -> the Lucene parser will automatically add it!
+    if (this.queryUrl && this.queryUrl.trim().substr(this.queryUrl.trim().length - 1) === '*') {
+      this.queryUrl = this.queryUrl.substring(0, this.queryUrl.trim().length - 1);
```

Comment:
Seems like it might be useful to move this `.trim()` up to line 177, just to eliminate a little repetition.

Reply:
Yep! Will do!